package.json

```diff
@@ -15,12 +15,12 @@
   ],
   "repository": {
     "type": "git",
-    "url": "git@github.com:canonical-web-and-design/react-components.git"
+    "url": "git@github.com:canonical/react-components.git"
   },
   "bugs": {
-    "url": "https://github.com/canonical-web-and-design/react-components/issues"
+    "url": "https://github.com/canonical/react-components/issues"
   },
-  "homepage": "/react-components/",
+  "homepage": "https://github.com/canonical/react-components",
```

Comment:
I think we had it as `/react-components` because Storybook uses it as a link under logo. So to make it work with GitHub-pages-hosted storybook we used to link to relative `/react-components`.

Maybe best to link it to `https://canonical.github.io/react-components` then? (one downside is that it will link to it in dev environment as well which may be confusing, but still better than broken link I guess).

Reply:
Good point, we're already linking to the repo from NPM, so `https://canonical.github.io/react-components/` would work better for the homepage. Will update.

BTW, it seems that the logo links to https://vanillaframework.io/ at the moment.